openhands/runtime/impl/eventstream/eventstream_runtime.py

```diff
@@ -332,7 +332,7 @@ def _init_container(
                 f'Error: Instance {self.container_name} FAILED to start container!\n'
             )
             logger.exception(e)
-            self.close(close_client=False)
+            self.close()
```

Comment:
The docker client is now shared due to use of lru_cache - so we no longer close it.